test/FxAccountsPairingChannel.js

```diff
@@ -23,44 +57,65 @@ describe('FxAccountsPairingChannel', () => {
         });
       });
       const {channelId, channelKey} = server;
-      const client = await FxAccountsPairingChannel.InsecurePairingChannel.connect(CHANNEL_SERVER, channelId, channelKey);
-      client.addEventListener('message', ({detail: {data}}) => {
-        CLIENT_RECV.push(data.msg);
+      const client = await FxAccountsPairingChannel.PairingChannel.connect(CHANNEL_SERVER, channelId, channelKey);
```

Comment:
You run against a network-based service in test?  That seems brittle.  Can you run this server locally? Or at least a cut-down version of the same.